BlueprintUICommonControls/Sources/AttributedLabel+Environment.swift

```diff
@@ -17,19 +17,20 @@ public protocol URLHandler {
     func onTap(url: URL)
 }
 
+@available(iOSApplicationExtension, unavailable, message: "This cannot be used in application extensions")
```

Comment:
Does this work?
```swift
private class NullURLHandler: URLHandler {
    func onTap(url: URL) { }
}

private final class DefaultURLHandler: NullURLHandler {
    @available(iOSApplicationExtension, unavailable)
    override func onTap(url: URL) {
        UIApplication.shared.open(url, options: [:], completionHandler: nil)
    }
}
```

Reply:
Assuming that the compiler allowed it (I don't think so), it would still actually trigger the URL in a share extension ~> App Store rejection

Reply:
OK this does actually work hah.
There's just the caveat that you should _always_ provide a url handler in an extension if you expect a URL

Reply:
Nice!